server/sonar-server/src/main/java/org/sonar/server/project/ws/BulkDeleteAction.java

```diff
@@ -67,20 +71,31 @@ public void define(WebService.NewController context) {
       .createParam(PARAM_KEYS)
       .setDescription("List of project keys to delete")
       .setExampleValue(KEY_PROJECT_EXAMPLE_001);
+
+    support.addOrganizationParam(action);
```

Comment:
I prefer discussing with @bellingard and lambda about introducing the term "root" into description of public web services.